server/config/passport.js

```diff
@@ -0,0 +1,26 @@
+import passport from 'passport';
+import bcrypt from 'bcrypt';
+import passportLo from 'passport-local';
+
+import AuthHelpers from '../helpers/authHelpers';
+
+const LocalStrategy = passportLo.Strategy;
+
+passport.use(new LocalStrategy({
+  usernameField: 'email',
+  passwordField: 'password',
+}, async (email, password, done) => {
+  try {
+    const userDocument = await AuthHelpers.userExists('email', email);
```

Comment:
Consider renaming the variable from **userDocument** to something like **user** or **userExists**

Reply:
It's done renamed to user